docs/networking/clusternetwork.md

```diff
@@ -78,14 +78,111 @@ A cluster network called `mgmt` is automatically created when a Harvester cluste
+
+The `mgmt` network uses the default MTU 1500 when you do not input another value other than 0 or 1500 for [management interface](../install/harvester-configuration.md#installmanagement_interface) while installing the cluster.
+
+When a non-default MTU value is set, follow [Annotate the non-default MTU value on mgmt Network after Installation](#annotate-the-non-default-mtu-value-on-mgmt-network-after-installation) right after the cluster is up.
+
+:::caution
+
+- Certain [ARP settings](https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt) can break cluster communications. With `arp_ignore=2`, for example, replies are sent only if the sender IP address is in the same subnet as the target IP address for which the MAC address is requested. This is not the case in a Harvester cluster, so using `arp_ignore=2` on all interfaces results in failed connectivity checks and prevents Longhorn pods (specifically, `backing-image` and `instance-manager`) from transitioning to the `Ready` state. Volumes cannot be attached to virtual machines if these Longhorn pods are not ready.
+
+- Due to the limitations on installation stage, you need to ensure each node to have the same MTU value, Harvester is not able to detect the differences on MTU values when a new node is joining the cluster. Different MTU values might cause unexpected results.
+
+:::
+
+#### Annotate the non-default MTU value on mgmt Network after Installation
+
+_Available as of v1.6.0_
+
+When your cluster is installed with a non-default MTU value like 9000, you need to annotate this value to the `mgmt` clusternetwork object. All the following created [VM network](./harvester-network.md#create-a-vm-network) inheriates this annotated MTU value automatically. Without it, the created `VM network` still defaults the MTU value to 1500.
+
+```
+
+$ kubectl annotate clusternetwork mgmt network.harvesterhci.io/uplink-mtu="9000"
+
+```
 
 :::caution
 
-Certain [ARP settings](https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt) can break cluster communications. With `arp_ignore=2`, for example, replies are sent only if the sender IP address is in the same subnet as the target IP address for which the MAC address is requested. This is not the case in a Harvester cluster, so using `arp_ignore=2` on all interfaces results in failed connectivity checks and prevents Longhorn pods (specifically, `backing-image` and `instance-manager`) from transitioning to the `Ready` state. Volumes cannot be attached to virtual machines if these Longhorn pods are not ready.
+- You need to ensure the annotation MTU value is same with the MTU value on [management interface](../install/harvester-configuration.md#installmanagement_interface) while installing the cluster.
+
+- You need to ensure all nodes are having the same MTU values.
+
+- For the default MTU value 1500, it does not matter to annotate or not.
```

Comment:
nit: might be simpler to say:
`You don't need annotation if you are using the default MTU value of 1500.`

also follows with above sentence and tense- You need/don't need/ing